awx/main/migrations/0199_alter_oauth2application_unique_together_and_more.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2.10 on 2024-10-24 14:06
```

Comment:
This migration file should be renamed to the obvious like "remove oauth2 models"